db/migrate/20190318130015_change_book_title_length.rb

```diff
@@ -0,0 +1,10 @@
+class ChangeBookTitleLength < ActiveRecord::Migration[5.1]
+  def change
+    Book.find_each do |book|
+      if book.title.length > 25
+        book.update(title: book.title.truncate(25))
```

Comment:
Beware about this logic, you dont have to truncate the title to 25 characters, If the title surpass 25 chars it should be 22 chars + '...'

Reply:
Example: 
```
Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Donec quam
```
This paragraph has 202 characters which is far from the limit of 25, it should be replaced with:
```
Lorem ipsum dolor sit ...
```

Reply:
.truncate(n) does exactly that.